rust/sealevel/environments/mainnet3/warp-routes/SOLX-nitro/token-config.json

```diff
@@ -9,6 +9,16 @@
     "interchainGasPaymaster": "AkeHBbE5JkwVppujCQQ6WuxsVsJtruBAjUo6fDCFp6fF",
     "owner": "BNGDJ1h9brgt6FFVd8No1TVAH48Fp44d7jkuydr1URwJ"
   },
+  "solaxy": {
+    "type": "synthetic",
+    "decimals": 6,
```

Comment:
as a consequence of this being a native warp route we'll need to have the local decimals be 9. Remote decimals stays at 18 though